.kokoro/samples-test.sh

```diff
@@ -37,6 +37,7 @@ npm install
 cd samples/
 npm link ../
 npm install
+npm install canvas
```

Comment:
These files are consistent across all of our repos - this wouldn't be the place to install `canvas` :) It belongs in `samples/package.json`.

Reply:
Fixed in latest commit.